packages/sw-routing/src/lib/error-factory.js

```diff
@@ -16,9 +16,9 @@
 import ErrorFactory from '../../../../lib/error-factory';
 
 const errors = {
-  'express-route-requires-absolute-path': `When using ExpressRoute, you must
-    provide a path that starts with a '/' character. You can only match
-    same-origin requests. For more flexibility, use RegExpRoute.`,
+  'express-route-invalid-path': `When using ExpressRoute, you must
+    provide a path that starts with a '/' character (to match same-origin
+    requests) or that starts with 'https://' (to match cross-origin requests)`,
```

Comment:
I've updated the message to reflect the fact that we're going to support `http:` URLs.

This is thrown when you try to instantiate an `ExpressRoute` using an invalid `path`. I am going to keep it verbose because the information is useful in this context—if someone passed in a bad `path`, I want to give them as much relevant information as I can so that they can figure out what they should pass in instead.